src/hb-algs.hh

```diff
@@ -645,16 +645,26 @@ hb_unsigned_mul_overflows (unsigned int count, unsigned int size)
   return (size > 0) && (count >= ((unsigned int) -1) / size);
 }
 
+static inline bool
+hb_signed_mul_overflows (int x, int y, int &result)
+{
 /* Right now we only have one use for signed overflow and as it
  * is GCC 5.1 > and clang we don't care about its fallback ATM */
 #ifndef __has_builtin
 # define __has_builtin(x) 0
 #endif
```

Comment:
Since this is leaving a general macro definition behind, move it outside this function, into hb.hh.

Reply:
done